content/outcomes/application-observability/tsm-app-troubleshooting-day2.md

```diff
@@ -0,0 +1,30 @@
+---
+date: '2021-07-13'
+lastmod: '2021-07-13'
+layout: single
+team:
+  - Mark Schweighardt
+  - Niran Even-Chen
+  - Andrew Huffman
+title: "Tanzu Service Mesh: Application Troubleshooting (day 2)"
+weight: 102
+---
+
+When a distributed application is experiencing issues it's not an easy task to figure out where to look. Tanzu service mesh provides views which can pinpoint faster where problems are coming from and reduce the time to identify the issue. One could start from the views we discussed in the day 1 section to see whether the application is experiencing issues and where the issues are, by looking at the "Golden metrics" and the dependencies between the different services. In addition, Tanzu service mesh provides more extensive views which can become helpful in the troubleshooting task. After identifying an area of possible cause within Tanzu service mesh, the operator would need to use an APM tool like Tanzu Observability to perform more extensive research into the root cause as the problem. APM tools have more capabilities when it comes to providing historical data, performing data analysis and bringing more information such as additional metrics and logs that may not exists in TSM or insights into additional layers that Tanzu service mesh does not touch. The following are the additional views that can be used for troubleshooting:
```

Comment:
- Say _Tanzu Service Mesh_ (capitalized)
- I'm not sure I would classify TO as an APM tool

Reply:
Fixed